xdp/src/tx_loop.rs

```diff
@@ -145,7 +145,7 @@ pub fn tx_loop<T: AsRef<[u8]>, A: AsRef<[SocketAddr]>>(
                 }
             }
             Err(TryRecvError::Empty) => {
-                if timeouts < MAX_TIMEOUTS {
+                if timeouts <= MAX_TIMEOUTS {
```

Comment:
This means we'll hold shreds up to 2ms, right?

Still seems fine, but want to make sure that was intentional

Reply:
fixed!

Reply:
save some blood for 🧠